### PR TITLE
bevy migration 0.10 -> 0.11 (tested all examples) macros syn 1.0 -> 2.0

### PR DESCRIPTION
**Pull request by GodGotzi**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,13 +11,16 @@ repository = "https://github.com/JonahPlusPlus/bevy_atmosphere"
 exclude = ["/assets/", "/examples/", "/.github/"]
 
 [dependencies]
-bevy = {version = "0.10", default-features = false, features = ["bevy_asset", "bevy_render", "bevy_pbr"]}
-bevy_atmosphere_macros = {path = "macros", version = "0.2"}
+bevy = {version = "0.11.0", default-features = false, features = ["bevy_asset", "bevy_render", "bevy_pbr"]}
+bevy_atmosphere_macros = { path = "macros" }
+bevy_spectator = "0.2.0"
```
Comment (Shatur) [inline Cargo.toml:16]: I think this crate should be only in dev-dependencies.

Comment (GodGotzi) [reply Cargo.toml:16]: yes you're right

**Comment by Shatur** [comment]
Please fix CI warnings

**Comment by GodGotzi** [comment]
The bevy_spectator isn't supporting 0.11 yet, so we need to wait till my pr gets merged.
Thats why I used my fork to get a version working with 0.11

File: macros/src/model.rs [hunk 2]
```diff
@@ -47,8 +47,8 @@ pub fn derive_atmospheric(ast: syn::DeriveInput) -> Result<TokenStream> {
     let atmosphere_path = super::bevy_atmosphere_path();
     let render_path = manifest.get_path("bevy_render");
     let asset_path = manifest.get_path("bevy_asset");
-    let app_path = manifest.get_path("bevy_app");
-
+    //let app_path = manifest.get_path("bevy_app");
```
Comment (Shatur) [inline macros/src/model.rs:50]: Why this line is commented out?

**Comment by Shatur** [comment]
Thanks! The original author is busy right now, so I will merge this and publish a new release.

**Pull request merged**